ihatemoney/templates/list_bills.html

```diff
@@ -123,7 +123,13 @@ <h3 class="modal-title">{{ _('Add a bill') }}</h3>
                 </td>
                 <td>{{ bill.payer }}</td>
                 <td>{{ bill.what }}</td>
-                <td>{{ bill.owers|join(', ', 'name') }} </td>
+                <td>{% if bill.owers|length == g.project.members|length -%}
+                {{ _("Everyone") }}
+                {%- elif bill.owers|length > g.project.members|length / 2 + 1 -%}
+                {{ _("Everyone excepted %(excluded)s", excluded=g.project.members|reject('in', bill.owers)|join(', ', 'name')) }}
```

Comment:
isn't it `except`? not sure about it

Reply:
I guess so. But I would go for "Everyone but […]" because it is shorter to read, and it is the goal of this change.

Reply:
« Everyone but » seems better to me :-)